.eslintrc.js

```diff
@@ -1,18 +1,19 @@
 module.exports = {
   env: {
     browser: true,
+    jest: true,
     es2021: true
   },
   extends: ['airbnb-base', 'plugin:prettier/recommended'],
   overrides: [],
-  plugins: ['unused-imports', 'import'],
+  // plugins: ['unused-imports', 'import'],
```

Comment:
add it back

Reply:
with this, I am not able to run eslint itself.
Getting errors.